src/kraft.erl

```diff
@@ -45,8 +74,58 @@ start(Opts, Routes) ->
     }),
     Ref.
 
-stop(Ref) -> kraft_instance:stop(Ref).
+%%------------------------------------------------------------------------------
+-doc """
+Stops the Kraft server instance.
+""".
+-spec stop(kraft_instance:ref()) -> ok.
+stop(Ref) ->
+    kraft_instance:stop(Ref).
+
+%%------------------------------------------------------------------------------
+-doc """
+Renders a template with the given context.
+
+This function allows you to render templates programmatically outside
+of the request/response cycle. Useful for generating emails, reports,
+or other content.
```

Comment:
I don't really understand this explanations. What is meant by "outside of the request/response cycle"?

I also think the suggested usefulness is not really accurate or helpful.